### PR TITLE
refactor: rename token to key

### PR DESCRIPTION
**Pull request by aifodu**
rename ApiToken to ApiKey and $apiToken to $apiKey across the project

Fixes issue #23

### CODE REVIEW

**Comment by nunomaduro** [comment]
@aifodu Tests are failing. Can you take a look?

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 *.swp
 *.swo
 playground/*
+.idea
```
Comment (Plytas) [inline .gitignore:10]: This should go into your global `.gitignore`

Comment (nunomaduro) [reply .gitignore:10]: @Plytas is right! @aifodu Can you remove this entry?

Comment (aifodu) [reply .gitignore:10]: Yes I can and I have, but I notice another request has already been made without any conflict to resolve this.

**Review by Plytas — CHANGES_REQUESTED** [review]
Whole `.idea` folder got committed now.
@nunomaduro there's also #25 that seems to be clean and with passing tests.

**Comment by aifodu** [comment]
Yes, #25 is clean and passes everything. Thanks @Plytas and @nunomaduro for taking a look. Tremendous job guys!

**Pull request closed without merging**